tools/scripts-locales/codeshift/locales.js

```diff
@@ -128,9 +128,15 @@ export default function transformer(fileInfo, api, options) {
 	try {
 		const en = getLocales(options.ref);
 		const j = api.jscodeshift;
-		const root = j(fileInfo.source);
-		let result = searchAndUpdateI18nValues(j, root, en);
-		if (result) {
+		options.comp = options.comp === 'all' ? ['i18n', 'trans'] : options.comp;
```

Comment:
it looks like options.comp is undefined by default.
Could you please make it 'all' by default

Reply:
fixed